### PR TITLE
[RELEASE] Enable snapshot release via Sonatype

### PR DESCRIPTION
**Pull request by changhiskhan**
See https://www.scala-sbt.org/1.x/docs/Using-Sonatype.html for details

### CODE REVIEW

File: version.sbt [hunk 1]
```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.0.1-SNAPSHOT"
```
Comment (eddyxu) [inline version.sbt:1]: Is this file be refered somewhere else

**Review by eddyxu — APPROVED** [review]
LGTM

File: build.sbt [hunk 2]
```diff
@@ -1,17 +1,29 @@
-organization := "ai.eto.rikai"
+scalaVersion := "2.12.11"
 
 name := "rikai"
 
-version := "0.0.1"
+organization := "ai.eto"
```
Comment (eddyxu) [inline build.sbt:5]: strange that this `organization` change did not break pyspark test

**Comment by eddyxu** [comment]
Mind to document the release procedure in README.md or wiki?

File: version.sbt (hunk 1 above)
Comment (changhiskhan) [reply version.sbt:1]: separate version.sbt is required for release automation via sbt-release (https://github.com/sbt/sbt-release)

**Comment by changhiskhan** [comment]
> Mind to document the release procedure in README.md or wiki?

I'll add it to the wiki separately. I don't think we need to add it to the README

**Pull request merged**